x-pack/plugins/aiops/public/hooks/use_time_buckets.ts

```diff
@@ -10,6 +10,7 @@ import { UI_SETTINGS } from '@kbn/data-plugin/common';
 import { TimeBuckets } from '../../common/time_buckets';
 import { useAiopsAppContext } from './use_aiops_app_context';
 
+// @TODO: refactor
```

Comment:
Note: Possibly worth refactoring this hook in a follow-up PR? The only difference between the plugins is the app context.